src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -152,6 +165,8 @@ export default {
 			showExpandableLoanCards: false,
 			rightArrowPosition: undefined,
 			leftArrowPosition: undefined,
+			hasFavoriteCountry: false,
+			favoriteCountryExpVersion: 'control',
```

Comment:
Not necessary to change but we could default this to `null` to increase readability